inference-engine/include/cldnn/cldnn_config.hpp

```diff
@@ -79,10 +79,12 @@ DECLARE_CLDNN_CONFIG_KEY(NV12_TWO_INPUTS);
 DECLARE_CLDNN_CONFIG_KEY(MAX_NUM_THREADS);
 
 /**
-* @brief This key enable loop unrolling for recurrent layers such as LSTM, GRU and TensorIterator.
-* By default the graph optimization unrolls TensorIterator layers.
-* If the ENABLE_LOOP_UNROLLING is turn off, TensorIterator body is separated to sub-program instead of loop unrolling,
-* compile it once, and then execute it with proper inputs and outputs for each iteration.
+* @brief Turning on this key enables to unroll recurrent layers such as TensorIterator or Loop with fixed iteration count.
+* This key is turned on by default. Note that unrolling loops will cause increase of clDNN build time,
+* because the kernels consiting the body network are created and built for all iterations.
+* Also, disabling unrolling for loops with small iteration count and small body network might perform worse than
+* unrolling, because the overhead for handling subnetworks might be more significant.
+* Thus this config should be set considering the iteration count and body network's size.
```

Comment:
What about stating more in a direct way? For example, `Thus the rule of thumb is to turn this option on only when iteration count is high and body network is large. In other words, only when loop overhead is negligible.`

Reply:
As we discussed offline, I will update these descriptions once the final performance is fixed.